fairscale/optim/oss.py

```diff
@@ -91,9 +91,9 @@ def partition_parameters(self) -> List[List[dict]]:
                     param_groups[rank].append(param_group_rank)
         return param_groups
 
-    def step(self, closure: Optional[Callable[[], float]] = None) -> Optional[float]:
+    def step(self, closure: Optional[Callable[[], float]] = None, **kwargs: Any) -> Optional[float]:
```

Comment:
Perhaps a few words here on the use case of kwargs for step() function? I haven't seen it used before. I am sure there is a valid use case. Perhaps useful to document the use case for future readers.